frontend/packages/ceph-storage-plugin/src/utils/dashboard.ts

```diff
@@ -1,15 +1,17 @@
-import * as _ from 'lodash';
 import { PrometheusResponse } from '@console/internal/components/graphs';
 
-export const getResiliencyProgress = (results: PrometheusResponse): number => {
+export const getResiliencyProgress = (
```

Comment:
when you are changing this, you need to change the places where this function is used